src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql

```diff
@@ -250,7 +250,7 @@ VALUES
  INSERT INTO Mtd.SourceConfigForImport
 (SourceDatabaseName,SourceTableName,SourceSchemaName,ColumnNamesToInclude,ColumnNamesToExclude,ColumnNamesToMask,ModelDataToPL,IsQueryBasedImport,SourceQuery,StagingTableName)
 VALUES
-   ('Assessor','Certificates','dbo','[Id],[StandardName],[StandardLevel],[StandardPublicationDate],[ContactOrganisation],[LearningStartDate],[AchievementDate],[CourseOption],[OverallGrade],[Department],[ToBePrinted],[CreatedAt],[DeletedAt],[OrganisationId],[BatchNumber],[Status],[UpdatedAt],[StandardCode],[IsPrivatelyFunded],[PrivatelyFundedStatus],[StandardUId],[EPADate],[Version]','[ContactAddLine1],[ContactAddLine2],[ContactAddLine3],[ContactAddLine4],[LearnRefNumber],[FullName],[LearnerGivenNames],[LearnerFamilyName],[ContactName]','[ContactPostCode],[Registration],[ProviderName],[StandardReference],[CertificateReference],[ProviderUkPrn],[CertificateReferenceId],[CreateDay],[Uln],[CreatedBy]',0,1
+   ('Assessor','Certificates','dbo','[Id],[StandardName],[StandardLevel],[StandardPublicationDate],[ContactOrganisation],[LearningStartDate],[AchievementDate],[CourseOption],[OverallGrade],[Department],[ToBePrinted],[CreatedAt],[DeletedAt],[OrganisationId],[BatchNumber],[Status],[UpdatedAt],[StandardCode],[IsPrivatelyFunded],[PrivatelyFundedStatus],[StandardUId],[EPADate],[Version]','[ContactAddLine1],[ContactAddLine2],[ContactAddLine3],[ContactAddLine4],[LearnRefNumber],[FullName],[LearnerGivenNames],[LearnerFamilyName],[ContactName],[UpdatedBy],[DeletedBy]','[ContactPostCode],[Registration],[ProviderName],[StandardReference],[CertificateReference],[ProviderUkPrn],[CertificateReferenceId],[CreateDay],[Uln],[CreatedBy]',0,1
```

Comment:
Did you check for all the tables to see if there are any columns missing to be in exclude list?